src/modules/transaction/components/TransactionRow/components.js

```diff
@@ -1,6 +1,6 @@
 import React, { useContext } from 'react';
 import { Link } from 'react-router-dom';
-import { getTransactionAmount } from '@transaction/utils/transaction';
+import { /* getTransactionAmount, */ getTransactionValue } from '@transaction/utils/transaction';
```

Comment:
Clean this and below comment `// import TransactionAmount from '../TransactionAmount';`